cmake/Modules/FindDetours.cmake

```diff
@@ -1,68 +1,75 @@
 # Once done these will be defined:
 #
-#  DETOURS_FOUND
-#  DETOURS_INCLUDE_DIRS
-#  DETOURS_LIBRARIES
+# DETOURS_FOUND
+# DETOURS_INCLUDE_DIRS
+# DETOURS_LIBRARIES
 #
 # For use in OBS:
 #
-#  DETOURS_INCLUDE_DIR
+# DETOURS_INCLUDE_DIR
```

Comment:
You can leave the indenting spaces in place until we have a good cmake-format config.

Reply:
Yeah didn't notice these - fixed.